doc/source/whatsnew/v0.23.1.txt

```diff
@@ -116,3 +116,4 @@ Other
 
 - Tab completion on :class:`Index` in IPython no longer outputs deprecation warnings (:issue:`21125`)
 - Bug preventing pandas from being importable with -OO optimization (:issue:`21071`)
+- Timestamp resolution returns a :class:`Timedelta` rather than a normal `timedelta` object (:issue:`21336`)
```

Comment:
I think the main point of the issue was that before the stated resolution was 1 microsecond while in actuality its 1 nanosecond (but we can't use `timedelta` to express that because it doesn't support nanoseconds)

Reply:
Resolved in commit c53f2c2